### ISSUE TITLE
It is awkward that this is both a template and instructions

### ISSUE THREAD

**Issue by remram44** [swcarpentry/lesson-example#164]
I think it is very useful to have a template lesson to easily create new lessons. However this repository also contains important information about creating/maintaining lesson in that example. I think those two things really ought to be broken apart, into a simple template/example, and a "maintainer lesson" (or added to [maintainer-onboarding](https://github.com/carpentries/maintainer-onboarding)?)

For example I was referred to the [maintenance page](http://swcarpentry.github.io/lesson-example/08-maintenance/index.html) that contains a lot of important info I don't think is elsewhere, and yet the pages are titled "Lesson Title".

**Comment by rgaiacs** [comment]
Hi @remram44, Thanks for the constructive feedback.

> However this repository also contains important information about creating/maintaining lessons. I think those two things really ought to be broken apart, into a simple template/example, and a "author/maintainer lesson" (or added to maintainer-onboarding?)

Some information are here because we didn't have a place to add it in a few months. @ErinBecker can move some content to https://github.com/carpentries/maintainer-onboarding as she is leading the maintainers onboarding.

**Comment by remram44** [comment]
No, this repository is still very much both example and instructions. Having important information about creating a lesson be under the heading "title here" is obviously wrong.

**Comment by bencomp** [comment]
Ah yes, I see your point. I thought you meant that this repository was the GitHub template repository to base lessons on, but I was wrong about that. I agree that "title here" is not helpful and even confusing when we point people to use this lesson as a (style) guide.
